docs/installation-and-operations/configuration/environment/README.md

```diff
@@ -61,82 +67,253 @@ Please note: The Configuration is a HashWithIndifferentAccess and thus it should
 
 # Supported environment variables
 
-Below is the full list of supported environment variables that can be used to override the default configuration of your OpenProject installation:
+A list of all configuration variables that can be overridden by defining them in the environment from bash could be generated by executing the following rake task:
+
+For packaged installations:
+
+```bash
+sudo openproject run bundle exec rake 'setting:available_envs'
+```
+
+For Docker installations:
 
+```bash
+docker exec -it $(docker ps | grep web | cut -d' ' -f 1) bash -c "RAILS_ENV=production bundle exec rake 'setting:available_envs'"
 ```
-OPENPROJECT_EDITION (default="standard")                                                                                                         
-OPENPROJECT_ATTACHMENTS__STORAGE (default="file")                                                                                                 
+
+The default value is also visible for each variable in parenthesis.
+
+```ruby
+#OpenProject Version 12.2.1
+
```

Comment:
I would suggest adding a comment here that the following are the correct values at the time of writing and that you should use the command shown above to make sure to get the up-to-date list.

Reply:
was added as suggested
@machisuji please check again